app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -108,7 +108,8 @@ fun MessageItem(
                     MessageContent(
                         messageContent = messageContent,
                         onAssetClick = currentOnAssetClicked,
-                        onImageClick = currentOnImageClick
+                        onImageClick = currentOnImageClick,
+                        onLongClick = { onLongClicked(message)}
```

Comment:
Won't that bring back the issue with flickering images when writing? That's why Mateusz used `remember`

Reply:
damn it u are right I think @Garzas  please check if anything gets recomposed unnecessary